src/mk.sh

```diff
@@ -525,6 +525,14 @@ function mk_build()
   IFS=' ' read -r -a options <<< "$raw_options"
   for option in "${options[@]}"; do
     case "$option" in
+      --info|-i)
+        local kernel_version=$(get_kernel_release "$flag")
+        local compiled_modules=$(cat modules.order | wc -l)
+        
+        echo -e "Release: $kernel_version"
+        echo -e "Total modules to be compiled: $compiled_modules"
+        exit
+        ;;
```

Comment:
I prefer to isolate the information code in a single function instead of adding it in the parser option. This can be useful for the following reasons:
1. It is easier to expand the build information feature;
2. We can have a dedicated test for this function.